tests/test-td-payload/src/testacpi.rs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+
+#![no_std]
+extern crate alloc;
+
+use crate::lib::{TestCase, TestResult};
+use alloc::string::{String, ToString};
+use alloc::vec::Vec;
+use core::ffi::c_void;
+use core::mem::size_of;
+use serde::{Deserialize, Serialize};
+use td_layout::memslice;
+use td_shim::acpi::GenericSdtHeader;
+use td_shim::TD_ACPI_TABLE_HOB_GUID;
+use td_uefi_pi::hob;
+use zerocopy::{AsBytes, FromBytes};
```

Comment:
Why we need this?

Reply:
GenericSdtHeader::read_from is used to extract the ACPI header from hob_list. It is derived from zerocopy::FromBytes. 
zerocopy::AsBytes is not needed. It will be removed.

Reply:
read_from is implemented in the trait of zerocopy::FromBytes. So we must import zerocopy::FromBytes in testacpi.rs.